### PR TITLE
Added /schema GET route

### PR DESCRIPTION
**Pull request by kuraisle**
Added the service for a /schema GET route, then added the route.

- #11

### CODE REVIEW

File: iog-api/iog_api/routes.py [hunk 1]
```diff
@@ -62,3 +65,6 @@ async def get_checks() -> List:
             "unique_values_eq"
             ]
 
+@router.get("/schema")
```
Comment (dcl10) [inline iog-api/iog_api/routes.py]:
```suggestion
@router.get("/schemas")
```

File: iog-api/iog_api/models.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from sqlalchemy import Column, Integer, String
 
-from db import Base
+from .db import Base
```
Comment (ATayls) [inline iog-api/iog_api/models.py]: from iog_api.db

File: iog-api/iog_api/services/schemas.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+from typing import Optional, List
+from sqlalchemy.orm import Session
+from pydantic import BaseModel, Json
+from ..models import Schemas
```
Comment (ATayls) [inline iog-api/iog_api/services/schemas.py:3]: from iog_api.models

**Review by dcl10 — CHANGES_REQUESTED** [review]

File: iog-api/test.db
Comment (dcl10) [inline iog-api/test.db:1]: Don't commit a database!

**Review by dcl10 — CHANGES_REQUESTED** [review]

**Review by dcl10 — APPROVED** [review]

**Review by ATayls — APPROVED** [review]

**Pull request merged**